src/v2/Apps/Conversation/Components/NoMessages.tsx

```diff
@@ -1,21 +1,30 @@
-import { Button, Flex, Sans } from "@artsy/palette"
+import { Box, Button, Flex, Sans } from "@artsy/palette"
 import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { EmptyInboxHeader } from "./InboxHeaders"
 import React from "react"
 
 export const NoMessages = () => {
   return (
-    <Flex
-      flexDirection="column"
-      justifyContent="center"
-      alignItems="center"
-      height="100%"
-    >
-      <Sans size="6" weight="medium" mb={2}>
-        You don't have any messages yet.
-      </Sans>
-      <RouterLink to="/collect">
-        <Button>Explore Artworks</Button>
-      </RouterLink>
-    </Flex>
+    <Box>
+      <EmptyInboxHeader />
+      <Flex
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        height="calc(100vh - 145px)"
```

Comment:
I wanted to go the height route, but there are _many_ parents that have to be modified to get that to work. It seems less tenable than vh (even if the latter isn't necessarily desirable itself). 

This might be one of those cases where we just have to shoot for good enough. Perhaps there's a percentage that works well enough?

Reply:
Agreed. Decided to shrink the empty inbox height for mobile for now in c873abc.